### PR TITLE
Theming: fix and run SASS autogenerator for theme slots

### PR DESCRIPTION
**Pull request by phkuo**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Fix the script that autogenerates the SASS variables for all the theme slots, then run it.

This is URGENT and blocking SharePoint evergreen because some semantic slots were incorrectly deleted.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12949)

### CODE REVIEW

**Comment by phkuo** [comment]
@msft-github-bot please autocomplete

File: change/@fluentui-react-next-2020-04-22-17-09-48-functional-checkbox.json [hunk 1]
```diff
@@ -1,8 +0,0 @@
-{
-  "type": "prerelease",
```
Comment (joschect) [inline change/@fluentui-react-next-2020-04-22-17-09-48-functional-checkbox.json:2]: I think you might have had a bad merge or pull, I don't think this should be deleted.

File: change/@uifabric-pr-deploy-site-2020-04-06-10-28-34-caperez-filemap-mar2020-fab7.json [hunk 2]
```diff
@@ -1,9 +0,0 @@
-{
-  "type": "patch",
-  "comment": "removing more references to wrong filetype icons from a deprecated folder",
```
Comment (joschect) [inline change/@uifabric-pr-deploy-site-2020-04-06-10-28-34-caperez-filemap-mar2020-fab7.json:3]: See above

**Comment by chrismohr** [comment]
I'm sorry, I think I missed the comment about it being an autogenerated file, and probably edited manually in https://github.com/microsoft/fluentui/pull/10296.

I didn't realize that the scss files get distributed.

Some additional documentation or tooling could probably help here. 

Seems that `yarn update-sass-theme-files` should be run after any theme slots change?  

If you think it would be useful, I could take a stab at automating a check.  Let me know.

**Comment by phkuo** [comment]
@chrismohr more automation would be great! Yeah, this has been a long-standing problem, and this isn't the first time we've forgotten to run the script...

**Pull request merged**